### PR TITLE
Added the fix for creating the multiple entries with single click

### PR DESCRIPTION
**Pull request by gautam071**
Targets #1351

### Description

- When a user clicks on the create button on the invoice page multiple times. The record is saved multiple times. Added the fix for this issue in this PR.
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by dsajwan2020** [comment]
Hi @GauravGusain98 can you please do the code review for this PR

**Review by GauravGusain98 — APPROVED** [review]
Looks fine.

File: Modules/Client/Resources/views/subviews/create-client-details.blade.php [hunk 1]
```diff
@@ -59,5 +59,17 @@ class="checkbox-custom mb-1.9 mb-1.67 mr-3"
...
+    const saveClient = (button) => {
+        button.disabled = true;
+        if (!button.form.checkValidity()) {
+            button.disabled = false;
+            button.form.reportValidity();
+            return;
+        }
+        button.form.submit();
+    }
+</script>
```
Comment (GauravGusain98) [inline Modules/Client/Resources/views/subviews/create-client-details.blade.php:75]: I see that this functionality is being used at multiple place. can we make a general function for this .You may include that function inside the main app.js file and then you just need to call that function in the button.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Just a minor change requested.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**